src/main/resources/modals.addServant/AddServantModal.fxml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.Font?>
+<AnchorPane prefHeight="216.0" prefWidth="302.0" styleClass="root" stylesheets="@AddServantStyles.css"
+            xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
```

Comment:
javafx version wrong!

Reply:
It was Gradle not me =(

Reply:
Wait no, I don't even know who did this.